### PR TITLE
Add embedding_renorm code | feat(torchlib)

### PR DESCRIPTION
**Pull request by xiaowuhu**

### CODE REVIEW

File: onnxscript/function_libs/torch_lib/ops/core.py [hunk 1]
```diff
@@ -3000,6 +3000,57 @@ def aten_embedding_dense_backward(
...
+@torch_op("aten::embedding_renorm", trace_only=True)
+def aten_embedding_renorm(
+    weight: TFloat, indices: INT64, max_norm: float, norm_type: float = 2.0
+) -> TFloat:
+    """embedding_renorm(Tensor weight, Tensor indices, float max_norm, float norm_type) -> Tensor"""
+
+    unique_indices = op.Unique(indices)
+    unique_indices_Y = op.SequenceAt(unique_indices, 0)
+    # using _onnx private function because op.SrquenceAt(unique_indices, 0) cannot pass module checker
+    # The error message is:
```
Comment (justinchuby) [inline onnxscript/function_libs/torch_lib/ops/core.py:3012]: Should we report this to ONNX?

File: onnxscript/function_libs/torch_lib/ops/core.py [hunk 2]
```diff
@@ -3000,6 +3000,57 @@ def aten_embedding_dense_backward(
...
+    # onnx.onnx_cpp2py_export.shape_inference.InferenceError:
+    # [ShapeInferenceError] Shape inference error(s): (op_type:aten_embedding_renorm,
+    # node name: aten_embedding_renorm_0): [ShapeInferenceError] (op_type:SequenceAt,
+    # node name: n2): input_sequence typestr: S, has unsupported type: tensor(int64)
+    return aten_embedding_renorm_onnx(weight, unique_indices_Y, max_norm, norm_type)
+
+
+@torch_op("aten::embedding_renorm", private=True)
+def aten_embedding_renorm_onnx(
+    weight: TFloat, indices: INT64, max_norm: float, norm_type: float = 2.0
```
[the remark this answers is not in the document]
Comment (xiaowuhu) [reply onnxscript/function_libs/torch_lib/ops/core.py:3022]: It is float in https://github.com/pytorch/pytorch/blob/330db8278b3d3eeef47237fe0d2411ff0ffcfab7/aten/src/ATen/native/native_functions.yaml#L2228C19-L2228C19

File: onnxscript/function_libs/torch_lib/ops/core.py (hunk 1 above)
Comment (xiaowuhu) [reply onnxscript/function_libs/torch_lib/ops/core.py:3012]: Filed an issue: https://github.com/onnx/onnx/issues/5698

**Review by justinchuby — APPROVED** [review]
🎆 Thank you!

File: onnxscript/function_libs/torch_lib/ops/core.py [hunk 3]
```diff
@@ -3000,6 +3000,57 @@ def aten_embedding_dense_backward(
...
 
 
+@torch_op("aten::embedding_renorm", trace_only=True)
+def aten_embedding_renorm(
+    weight: TFloat, indices: INT64, max_norm: float, norm_type: float = 2.0
+) -> TFloat:
+    """embedding_renorm(Tensor weight, Tensor indices, float max_norm, float norm_type) -> Tensor"""
+
+    unique_indices = op.Unique(indices)
+    unique_indices_Y = op.SequenceAt(unique_indices, 0)
```
Comment (justinchuby) [inline onnxscript/function_libs/torch_lib/ops/core.py:3010]: Should we use slice here?

Comment (xiaowuhu) [reply onnxscript/function_libs/torch_lib/ops/core.py:3010]: Nop. It is an unequal length list.

**Pull request merged**